overlays/ci.yml

```diff
@@ -12,6 +12,33 @@ applications:
   mariadb:
     options:
       root-password: foobar
+  modeldb-backend:
+    options:
+      config: |
```

Comment:
@johnsca: Wishlist item for new charming - ability to define something like this:

```yaml
options:
  database-password:
    type: string
    default: ''
    description: Password to use for database connection
  config.yaml:
    type: template
    description: /config/config.yaml
    default: |
      database:
        username: root
        password: {{ database_password }}
```

That way we wouldn't have to copy this entire block just to change the `RdbPassword` bit, but could allow someone to completely override the config if they needed to (unlike if we have the charm's Python code render the template)